src/pages/ir-replineshment-grps/forms/ReplineshmentForm.js

```diff
@@ -0,0 +1,175 @@
+import { Grid } from '@mui/material'
+import { useContext, useEffect } from 'react'
+import * as yup from 'yup'
+import FormShell from 'src/components/Shared/FormShell'
+import toast from 'react-hot-toast'
+import { RequestsContext } from 'src/providers/RequestsContext'
+import { useInvalidate } from 'src/hooks/resource'
+import { ResourceIds } from 'src/resources/ResourceIds'
+import CustomTextField from 'src/components/Inputs/CustomTextField'
+import { useForm } from 'src/hooks/form'
+import { Grow } from 'src/components/Shared/Layouts/Grow'
+import { VertLayout } from 'src/components/Shared/Layouts/VertLayout'
+import { ControlContext } from 'src/providers/ControlContext'
+import CustomNumberField from 'src/components/Inputs/CustomNumberField'
+import { IVReplenishementRepository } from 'src/repositories/IVReplenishementRepository'
+
+export default function ReplineshmentForm({ labels, maxAccess, recordId, invalidate }) {
+  const { getRequest, postRequest } = useContext(RequestsContext)
+  const { platformLabels } = useContext(ControlContext)
+
+  const { formik } = useForm({
+    initialValues: {
+      recordId: recordId || null,
+      reference: '',
+      name: '',
+      defaultMaxQty: '',
+      defaultMinQty: ''
+    },
+    enableReinitialize: true,
+    validateOnChange: true,
+    validationSchema: yup.object({
+      name: yup.string().required(),
+      reference: yup.string().required(),
+      defaultMinQty: yup
+        .number()
+        .nullable()
+        .min(0, 'Must be greater than or equal to 0')
```

Comment:
should this be included in labels or keep it hardcoded?
and all other conditions same

Reply:
those are invisible

Reply:
if the condition is false they should appear i guess!